tests/MongoDB.Driver.Legacy.Tests/Builders/IndexKeysBuilderTypedTests.cs

```diff
@@ -89,7 +89,14 @@ public void CreateIndex_with_wildcardProjection_should_create_expected_index()
             var indexes = collection.GetIndexes();
             var index = indexes.RawDocuments.Single(i => i["name"].AsString == "custom");
             index["key"]["$**"].AsInt32.Should().Be(1);
-            index["wildcardProjection"].ToBsonDocument().Should().Be(BsonDocument.Parse("{ b : 1, _id : 0 }"));
+            if (CoreTestConfiguration.ServerVersion >= new SemanticVersion(4, 5, 0, ""))
+            {
+                index["wildcardProjection"].ToBsonDocument().Should().Be(BsonDocument.Parse("{ b : true, _id : false }"));
```

Comment:
just minor suggestion, you can move this code:

    index["wildcardProjection"].ToBsonDocument()

 into the separate variable before `if`

Reply:
I've noticed redundant `.ToBsonDocument()` right after pushing the code, so I force pushed. Could you please recheck?